app/models/canonical_email_block.rb

```diff
@@ -26,7 +26,7 @@ def to_log_human_identifier
   end
 
   def email=(email)
-    self.canonical_email_hash = email_to_canonical_email_hash(email)
+    self.canonical_email_hash = self.class.email_to_canonical_email_hash(email)
```

Comment:
As a module, we were previously getting both methods as BOTH instance and class methods (instance via the inclusion, class b/c the module `included` wrapper). As a model concern I've made them both class methods, hence this change.